katello_certs_tools/fileutils.py

```diff
@@ -79,17 +80,15 @@ def rotateFile(filepath, depth=5, suffix='.', verbosity=0):
     """
 
     # check argument sanity (should really be down outside of this function)
-    if not filepath or type(filepath) != type(''):
+    if not filepath or not isinstance(filepath, str):
         raise ValueError("filepath '%s' is not a valid arguement" % filepath)
-    if type(depth) != type(0) or depth < -1 \
-      or depth > sys.maxint-1 or depth == 0:
+    if not isinstance(depth, int) or depth < -1 or depth > sys.maxint-1 or depth == 0:
```

Comment:
When is depth actually sys.maxint? Isn't that very unlikely? Especially since the parameter is never passed in. I'm noting this because sys doesn't have a maxint attribute in Python 3.

Reply:
No idea. I've never tried to understand this code. Just keep it alive :)